.packit.yaml

```
@@ -0,0 +1,40 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman_statistics.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman_statistics.spec
+    - .packit.yaml
```

I like consistent indenting.
```suggestion
  - rubygem-foreman_statistics.spec
  - .packit.yaml
```